src/EventMeta.php

```diff
@@ -608,4 +610,64 @@ protected function entityTypeHasBundles($entity_type_id) {
     return ($entity_type->getBundleEntityType() !== NULL);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  function createDefaultEventMessages() {
+    // Get Default messages for this Event type.
+    $default_messages = $this->getEventType()->getDefaultMessages();
+    if ($default_messages) {
+      foreach ($default_messages as $default_message) {
+        // Create Event Messages from Default Messages.
+        /** @var \Drupal\courier\Service\CourierManagerInterface $courier_manager */
+        $courier_manager = \Drupal::service('courier.manager');
```

Comment:
Why you use these services via static method instead using Dependency Injection?

Reply:
fixed, plz re-check